PluginsMakefile.mk

```diff
@@ -103,7 +115,14 @@ npm: ## Run a npm command, example: make npm c='install mypackage/package'
 .PHONY: npm
 
 ##—— Testing and static analysis ———————————————————————————————————————————————
-verify: license-headers-check parallel-lint phpstan phpcsfixer-check phpunit ## Run all our lints/tests/static analysis
+verify:  ## Run all our lints/tests/static analysis
```

Comment:
I've been looking online to see if there is a consensus regarding what the name of a command that run all the testing/linting/analysis tool should be.

It seems `make test` is a bit more standard (but I do see some references to `make verify` too).
I think it would be best to use the most "standard" name, we can keep `verify` as an alias if you want so both are supported.

Reply:
I agree, make test will be more transparent for developers. we can keep `verify`as an alias (I'll be more inclined to remove but it's up to you)